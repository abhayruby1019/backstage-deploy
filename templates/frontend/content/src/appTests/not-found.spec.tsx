import { render } from '@testing-library/react'
import NotFound from '@/app/not-found'

describe('NotFound', () => {
  it('should display technical error when type is not passed', () => {
    const { container } = render(<NotFound />)
    expect(container).toMatchSnapshot()
  })
})
