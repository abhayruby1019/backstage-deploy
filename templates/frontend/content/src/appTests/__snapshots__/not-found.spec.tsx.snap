// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should display technical error when type is not passed 1`] = `
<div>
  <div
    class="mainError"
  >
    <div
      class="title"
    >
      Page Not Found
    </div>
    <a
      class="link"
      href="/"
    >
      Return Home
    </a>
  </div>
</div>
`;
